<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPedidosFiltro</class>
 <widget class="QDialog" name="dlgPedidosFiltro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtrar Pedidos</string>
  </property>
  <widget class="QDialogButtonBox" name="bboxFiltrarPedidos">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>96</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="deFechaDesde">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>12</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="deFechaHasta">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>42</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>14</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Fecha Desde:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>44</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Fecha Hasta:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bboxFiltrarPedidos</sender>
   <signal>accepted()</signal>
   <receiver>dlgPedidosFiltro</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bboxFiltrarPedidos</sender>
   <signal>rejected()</signal>
   <receiver>dlgPedidosFiltro</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
